templates/patroni.service.j2

```diff
@@ -0,0 +1,22 @@
+#########################################################################################
+# [ WARNING ]
+# patroni service unit file maintained by the postgres-operator
+# local changes may be overwritten.
+#########################################################################################
+[Unit]
+Description=Runners to orchestrate a high-availability PostgreSQL
+After=syslog.target network.target
+
+[Service]
+Type=simple
+
+User=postgres
+Group=postgres
+
+ExecStart=/usr/local/bin/patroni {{ conf_path }}/patroni.yml
+KillMode=process
+TimeoutSec=30
+Restart=no
+
+[Install]
+WantedBy=multi-user.targ
```

Comment:
```suggestion
WantedBy=multi-user.target
```

Is this a typo? :)

Reply:
Yes, you're right Jon. Thanks for catching it.